os/osx/app.mm

```diff
@@ -52,7 +52,14 @@ void setAppMode(AppMode appMode)
 
   void finishLaunching()
   {
-    [m_app finishLaunching];
+    // [m_app run] can only be called once.
+    if (![[NSRunningApplication currentApplication] isFinishedLaunching])
+      // Note that the [m_app run] call doesn't block because we are calling
+      // [NSApp stop] from [AppDelegateOSX applicationDidFinishLaunching]. We only
+      // need the application's initialization done inside run to prevent issues
+      // such as: https://github.com/aseprite/aseprite/issues/4795
+      [m_app run];
+
     [m_appDelegate resetCliFiles];
```

Comment:
Should we put this line under the scope of the if clause as well?

Reply:
Just in case, we should test if dropping files to the app icon in the dock works. Probably it should be tested with the final Aseprite.app bundle, not only the `bin/aseprite` binary. I could give a try next week.

Reply:
I've tested with the binary with and without this fix, and it doesn't work either way. So it seems that we can only test this with the final bundle?

Reply:
I've just kind of tested this by vandalizing a v1.3.12 bundle (replaced the release binary with a compiled one 😬) and it worked...not sure if it will work by creating the bundle, but seems like it.